Scheduler/observing_blocks_maintel/AOS/LUTs/BLOCK-T91.json

```diff
@@ -0,0 +1,212 @@
+        "note": "LUT_rotator_angle"
+      }
+    },
+    {
+      "name": "maintel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "track_azel": {
+          "az": "$run_azimuth",
+          "el": "$run_elevation"
+        },
+        "rot_value": -21.25,
+        "rot_type": "PhysicalSky"
+      }
+    },
+    {
+      "name": "maintel/take_triplet_comcam.py",
+      "standard": true,
+      "parameters": {
+        "filter": "$run_filter",
+        "program": "$program",
+        "reason": "SITCOM-1491",
+        "note": "LUT_rotator_angle"
+      }
+    },
+    {
+      "name": "maintel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "track_azel": {
+          "az": "$run_azimuth",
+          "el": "$run_elevation"
+        },
+        "rot_value": 0.0,
+        "rot_type": "PhysicalSky"
+      }
+    },
+    {
+      "name": "maintel/take_triplet_comcam.py",
+      "standard": true,
+      "parameters": {
+        "filter": "$run_filter",
+        "program": "$program",
+        "reason": "SITCOM-1491",
+        "note": "LUT_rotator_angle"
+      }
+    },
+    {
+      "name": "maintel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "track_azel": {
+          "az": "$run_azimuth",
+          "el": "$run_elevation"
+        },
+        "rot_value": 21.25,
+        "rot_type": "PhysicalSky"
+      }
+    },
+    {
+      "name": "maintel/take_triplet_comcam.py",
+      "standard": true,
+      "parameters": {
+        "filter": "$run_filter",
+        "program": "$program",
+        "reason": "SITCOM-1491",
+        "note": "LUT_rotator_angle"
+      }
+    },
+    {
+      "name": "maintel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "track_azel": {
+          "az": "$run_azimuth",
+          "el": "$run_elevation"
+        },
+        "rot_value": 42.5,
+        "rot_type": "PhysicalSky"
+      }
+    },
+    {
+      "name": "maintel/take_triplet_comcam.py",
+      "standard": true,
+      "parameters": {
+        "filter": "$run_filter",
+        "program": "$program",
+        "reason": "SITCOM-1491",
+        "note": "LUT_rotator_angle"
+      }
+    },
+    {
+      "name": "maintel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "track_azel": {
+          "az": "$run_azimuth",
+          "el": "$run_elevation"
+        },
+        "rot_value": 63.75,
+        "rot_type": "PhysicalSky"
+      }
+    },
+    {
+      "name": "maintel/take_triplet_comcam.py",
+      "standard": true,
+      "parameters": {
+        "filter": "$run_filter",
+        "program": "$program",
+        "reason": "SITCOM-1491",
+        "note": "LUT_rotator_angle"
+      }
+    },
+    {
+      "name": "maintel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "track_azel": {
+          "az": "$run_azimuth",
+          "el": "$run_elevation"
+        },
+        "rot_value": 85.0,
```

Comment:
Current rotator limits are -80.0 to +80.0 (see here https://github.com/lsst-ts/ts_cycle_build/blob/1cb648f29715c23a0f401bccdb58ca0d9ad0b511/build/ptg/MTPtg.info#L38).

Please update this to be within the limit and add a small buffer for tracking e.g. 75.0